src/storybook.test.js

```diff
@@ -0,0 +1,17 @@
+import initStoryshots from '@storybook/addon-storyshots';
+
+Object.defineProperty(window, 'matchMedia', {
```

Comment:
Can we break the snapshots based on stories, otherwise we'll get into lot of conflicts

Reply:
The snapshots are managed by the plugin, it maintains the order, etc. to avoid the conflicts.